samples/ingestion/ingestion-client/FetchTranscription/FetchTranscription.csproj

```diff
@@ -17,6 +17,7 @@
   </PropertyGroup>  
   <ItemGroup>
     <PackageReference Include="Azure.AI.TextAnalytics" Version="5.1.0" />
+    <PackageReference Include="Azure.Messaging.ServiceBus" Version="7.5.0" />
     <PackageReference Include="Microsoft.Azure.WebJobs.Extensions.ServiceBus" Version="4.3.0" />
```

Comment:
The Microsoft.Azure.WebJobs.Extensions.ServiceBus dependency needs to be updated to 5.1.0.